sdk/communication/communication-call-automation/src/models/options.ts

```diff
@@ -342,3 +342,15 @@ export interface StopTranscriptionOptions extends OperationOptions {
   /** The value to identify context of the operation. */
   operationContext?: string;
 }
+
+/**
+ * Options to hold participant.
+ */
+export interface HoldOptions extends OperationOptions {
```

Comment:
Create an `UnholdOptions` as well, so you push any optional parameter there.

Reply:
Resolved.